tools/mate-color-select.desktop.in

```diff
@@ -1,7 +1,7 @@
 [Desktop Entry]
-_Name=MATE Color Selection
-_GenericName=Color selection dialog
-_Comment=Choose colors from the palette or the screen
```

Comment:
Same here, shouldn`t those lines in desktop files are be translatable?

Reply:
See above.

Reply:
good